cli/tests/test_cli.py

```
@@ -0,0 +1,1491 @@
+
+class TestQueryCommands:
+    """Test suite for query commands (get_folders, get_perimeters, get_matrices)"""
+    
+    def test_get_folders_success(self, mock_token):
+        """
+        Test successful retrieval of folders
+        Input: Mock API response with valid folders
+        Output: JSON of folders
+        Assertions: JSON contains mocked folders, correct format
+        """
+        mock_folders_response = {
```

Ruff (0.11.9)

Local variable `mock_folders_response` is assigned to but never used

Remove assignment to unused variable `mock_folders_response`

(F841)